exporter/awsemfexporter/emf_exporter.go

```diff
@@ -174,21 +174,23 @@ func generateLogEventFromMetric(metric pdata.Metrics, dimensionRollupOption stri
 	cwMetricLists := []*CWMetrics{}
 	var cwm []*CWMetrics
 	var totalDroppedMetrics int
+	cwMetricsMap := make(map[string]*CWMetrics)
+	groupedCWMetricMap := make(map[string]*GroupedCWMetric) 
 
 	for i := 0; i < rms.Len(); i++ {
 		rm := rms.At(i)
 		if rm.IsNil() {
 			continue
 		}
-		cwm, totalDroppedMetrics = TranslateOtToCWMetric(&rm, dimensionRollupOption, namespace)
+		cwm, totalDroppedMetrics = TranslateOtToCWMetric(&rm, dimensionRollupOption, cwMetricsMap, groupedCWMetricMap, namespace)
 		if len(cwm) > 0 && len(cwm[0].Measurements) > 0 {
 			namespace = cwm[0].Measurements[0].Namespace
 		}
 		// append all datapoint metrics in the request into CWMetric list
 		cwMetricLists = append(cwMetricLists, cwm...)
```

Comment:
These piece of code doesn't looks clean. do we still use `cwMetricLists`? I think we need to reconsider where to do the batch logic